cheatsheets/Protect_FileUpload_Against_Malicious_File.md

```diff
@@ -1,597 +1,138 @@
-# Introduction
+# File Upload Cheat Sheet
 
-This article propose a way to protect a file upload feature against submission of file containing malicious code.
+## Introduction
 
-# Context
+File upload is becoming more and more an essential part of any application, where the user is able to upload their photo, their CV, or a video showcasing a project they are working on. It is evident that applications should be able to handle file uploads based on the business requirement. In return, the application should be able to fend off bogus, and malicious files in a way to keep the application and the users safe.
```

Comment:
Should there be a reference here to the input validation cheetsheet, being that handling file uploads is a special case of input validation?

Reply:
Input validation is referenced across the document. The opposite should be done though properly 😄